pkgs/development/libraries/science/math/brial/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkgconfig
+, boost
+, m4ri
+, gd
+}:
+
+stdenv.mkDerivation rec {
+  version = "1.2.3";
+  name = "brial-${version}";
+
+  src = fetchFromGitHub {
+    owner = "BRiAl";
+    repo = "BRiAl";
+    rev = version;
+    sha256 = "0qy4cwy7qrk4zg151cmws5cglaa866z461cnj9wdnalabs7v7qbg";
+  };
+
+  buildInputs = [
+    boost
+    m4ri
+    gd
+  ];
+
+  configureFlags = [
+    "--with-boost-unit-test-framework=no" # boost-test is not packaged
```

Comment:
Testing would probably reduce the necessary maintenance effort. So if it isn't too difficult, I'd suggest packaging `boost-test` and adding `doCheck = true`.

Reply:
I've done that (`boost init` commit is now included in this PR).

Reply:
Thanks a lot!